assets/src/components/content/add_resource_content/AddResourceContent.tsx

```diff
@@ -1,8 +1,9 @@
 import { ActivityEditContext } from 'data/content/activity';
 import { ResourceContent } from 'data/content/resource';
 import React, { useState } from 'react';
-import { Popover } from 'react-tiny-popover';
+import { OverlayTrigger, Popover } from 'react-bootstrap';
```

Comment:
Change add resource content popup from react-tiny-tooltip to react-bootstrap popover. The react-tiny-tooltip popover has too many positioning bugs. Unfortunately with the react-bootstrap popover we lose the ability to track whether it's "open" or not without a lot of work, so the "active" styling doesn't apply to the button. But it fixes the positioning issues which I cannot get to work with the other library.